### PR TITLE
Update to abscissa 0.6.0-rc.0 and clap 3.0

### PR DESCRIPTION
**Pull request by mzabaluev**
Closes: #1752

## Description

Mostly trivial changes to update to the stable API of clap 3.0

______

### PR author checklist:
- [x] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules). 
- [x] Linked to GitHub issue.
- ~Updated code comments and documentation (e.g., `docs/`).~

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

**Comment by mzabaluev** [comment]
Waiting for the next release of modelator including https://github.com/informalsystems/modelator/pull/163 before this can be undrafted with a proper version dependency.

**Comment by mzabaluev** [comment]
The md-link-check failure is fixed by #1778.

File: relayer-cli/src/commands/keys.rs [hunk 1]
```diff
@@ -1,27 +1,24 @@
...
     Delete(delete::KeysDeleteCmd),
 
-    /// The `keys list` subcommand
-    #[clap(about = "List keys configured on a chain")]
+    /// List keys configured on a chain
     List(list::KeysListCmd),
 
-    /// The `keys restore` subcommand
-    #[clap(about = "restore a key to a configured chain using a mnemonic")]
+    /// restore a key to a configured chain using a mnemonic
```
Comment (romac) [inline relayer-cli/src/commands/keys.rs:22]:
```suggestion
    /// Restore a key to a configured chain using a mnemonic
```

**Review by romac — APPROVED** [review]
Thank you so much for doing this update :)

**Pull request merged**